components/ADempiere/Form/VAllocation/components/Invoce.vue

```diff
@@ -0,0 +1,268 @@
+<!--
+ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
+Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A.
+Contributor(s): Elsio Sanchez elsiosanches@gmail.com https://github.com/elsiosanchez
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <https:www.gnu.org/licenses/>.
+-->
+
+<template>
+  <div style="display: contents;height: 100% !important;">
+    <el-table
+      :data="tableData"
+      border
+      style="width: 100%"
+    >
+      <el-table-column
+        v-for="(header, key) in headers"
+        :key="key"
+        prop="id"
+        :type="header.type"
+        :label="header.label"
+      />
+    </el-table>
+    <el-card class="box-card">
+      <div slot="header" class="clearfix" style="text-align: center;">
+        <b>
+          {{ 'Detalles de la Factura' }}
+        </b>
+      </div>
+      <div id="description-payment" class="description-payment" style="display: flex;">
+        <el-card
+          class="box-card"
+          style="width: 50%;float: left;"
+        >
+          <div>
+            <el-form
+              :inline="true"
+              label-position="left"
+              style="padding: 10px !important;"
+            >
+              <el-col :span="24">
+                <el-form-item
+                  label="Cantidad de Registro:"
```

Comment:
@elsiosanchez please add translation.

Reply:
Added the translation in the following commit [68211fe](https://github.com/solop-develop/frontend-default-theme/pull/442/commits/68211fe355873ffe49725eb888013312491e1af4)